drivers/display/display_st7789v.c

```diff
@@ -409,11 +424,27 @@ static const struct display_driver_api st7789v_api = {
 
 static const struct st7789v_config st7789v_config = {
 	.bus = SPI_DT_SPEC_INST_GET(0, SPI_OP_MODE_MASTER | SPI_WORD_SET(8), 0),
+	.vcom = DT_INST_PROP(0, vcom),
+	.gctrl = DT_INST_PROP(0, gctrl),
+	.vdv_vrh_enable = (DT_INST_NODE_HAS_PROP(0, vrhs) && DT_INST_NODE_HAS_PROP(0, vdvs)),
+	.vrh_value = DT_INST_PROP(0, vrhs),
+	.vdv_value = DT_INST_PROP(0, vdvs),
```

Comment:
how does this work if `vrhs` or `vdvs` are not present? Are these boolean props?

Reply:
Good catch! These are integer properties.  I tested that the build fails if one of these is missing. I added a default value to fix it.